### PR TITLE
Neard update 1.32

### PR DESCRIPTION
**Pull request by yanganto**
@bors force push `neard-updates` every day, so this PR is opened here to avoid being overwritten by the bot before checking in testnet.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -36,3 +36,7 @@ path = "src/bin/kuutamod.rs"
 [[bin]]
 name = "kuutamoctl"
 path = "src/bin/kuutamoctl.rs"
+
+[features]
+# work with neard-unstable
```
Comment (Mic92) [inline Cargo.toml:41]:
```suggestion
# set this for neard-unstable
```

**Review by Mic92 — APPROVED** [review]

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
CI is flakey, CI admin is informed about this.

**Comment by yanganto** [comment]
bors cancel

**Comment by yanganto** [comment]
bors merge

**Pull request merged**